src/components/developed-by/developed-by.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+import { PartnerLogo } from "../partner-logo/partner-logo";
+import { DeveloperLogo } from "../developer-logo/developer-logo";
+
+import "./developed-by.scss";
+
+export const DevelopedBy = () => {
```

Comment:
I think we should accept children elements and be able to override the icons. I think we can keep these as default but also allow other logos to be passed down in the component.

Reply:
I can move the caption part inside to logo containers. That would allow better use of children prop. Is this the way you'd like to use the children ?

Reply:
Let's go with this for now, we can update it later.